test/fixtures/cookbooks/test/recipes/install.rb

```diff
@@ -2,7 +2,7 @@
 
 service 'grafana-server' do
   action [:enable, :start]
-  subscribes :restart, ['template[/etc/grafana/grafana.ini]', 'template[/etc/grafana/ldap.toml]'], :delayed
+  subscribes :restart, ['template[/etc/grafana/grafana.ini]', 'template[/etc/grafana/ldap.toml]'], :immediately
```

Comment:
You recommend in the upgrading guide to remove this resource, why keep it and just change it to immediately.
I can understand keeping it around for the enable, start to make sure at the end of the run the service is started so grafana comes up.